devfile.yaml

```diff
@@ -1,57 +1,21 @@
 apiVersion: 1.0.0
 metadata:
   name: che-server-in-che
-projects:
-  - name: che
-    source:
-      type: git
-      location: 'https://github.com/eclipse/che.git'
 components:
   - type: dockerimage
-    image: 'quay.io/eclipse/che-dev:nightly'
-    alias: che-server-dev
+    image: quay.io/eclipse/che-java11-maven:nightly
+    alias: maven
     mountSources: true
-    memoryLimit: 5Gi
+    memoryLimit: 2Gi
     volumes:
       - name: maven
         containerPath: /home/user/.m2
-  - type: dockerimage
-    image: 'quay.io/eclipse/che-server:nightly'
-    alias: che-server-runtime
-    mountSources: true
-    memoryLimit: 1Gi
-    endpoints:
-      - name: che-dev
-        attributes:
-          discoverable: 'true'
-          public: 'true'
-        port: 8080
-    env:
-      - name: CHE_HOST
-        value: 'che-dev'
-      - name: CHE_API
-        value: http://che-dev:8080/api
-      - name: CHE_API_INTERNAL
-        value: http://che-dev:8080/api
-      - name: CHE_API_EXTERNAL
-        value: http://che-dev:8080/api
-    command:
-      - 'tail'
-    args:
-      - '-f'
-      - '/dev/null'
   - type: chePlugin
-    id: redhat/java8/latest
+    id: redhat/java11/latest
 commands:
-  - name: build
-    actions:
-      - type: exec
-        component: che-server-dev
-        command: mvn clean install -Pnative -DskipIntegrationTests
```

Comment:
Is it possible to build Che with tests using a command now?

Reply:
not yet

Reply:
and it will not be done in this PR. We'll do it in next iteration